ocp_resources/utils.py

```diff
@@ -204,12 +204,7 @@ def _is_exception_matched(exp, exception_messages):
         if not exception_messages:
             return True
 
-        for msg in exception_messages:
-            # Prevent match if provided with empty string
```

Comment:
Can we please retain this comment though ?

Reply:
ACK